templates/web/base/auth/smsform.html

```diff
@@ -0,0 +1,34 @@
+[% INCLUDE 'header.html', bodyclass = 'fullwidthpage', title = loc('Confirm account') %]
+
+[% IF token_not_found %]
+
+  <div class="confirmation-header confirmation-header--failure">
+    <h1>[% loc('Sorry, that wasn&rsquo;t a valid link') %]</h1>
+    <p>[% loc('The link might have expired, or maybe you didn&rsquo;t quite copy and paste it correctly.') %]</p>
+  </div>
```

Comment:
This is just sharing code with the email-token; this should never conceivably happen, as the token was submitted along with the form, but if it did, something is seriously wrong (the only one a user should conceivably ever see is if they take longer than a day to try and enter a code!) rather than just the wrong code.